pycbc/mchirp_area.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+# mchirp_area.py
+# Integration of the area laying in the different cbc regions
+# By A. Curiel Barroso
+# August 2019
+
+"""This script computes the area corresponding
+to different CBC on the m1 & m2 plane when
+given a central mchirp value and uncertainty
+It also includes a function that calculates
+the source frame when given the detector
+frame mass and redshift
+"""
+
+from pycbc.conversions import mass2_from_mchirp_mass1 as m2mcm1
+from scipy.integrate import quad
+
+
+def src_mass_from_z_det_mass(z, del_z, mdet, del_mdet):
+    """This function takes values of redshift, redshift uncertainty,
+    detector mass and its uncertainty and computes the source mass
+    and its uncertainty.
+    """
+    msrc = (mdet) / (1 + z)
+    del_msrc = msrc * ((del_mdet/mdet)**2 + (del_z/(1 + z))**2)**(0.5)
```

Comment:
PEP8 issues not caught by codeclimate .. This should have more spacing - ideally spaces around each operator.  Also '1' and '2' should be '1.' and '2.' to force them to be floats. 
Also 'mdet' and '0.5' shouldn't be in brackets when they are on their own.